test/functional/Java8andUp/src/org/openj9/test/java/lang/Test_Thread.java

```diff
@@ -787,6 +787,7 @@ public void test_join() {
 			// }
 			th.join();
 		} catch (InterruptedException e) {
+			AssertJUnit.fail("Interrupted", e);
 			result = false;
```

Comment:
The local variable `result` becomes useless with this change. Unless we expect to revert this later, the local variables should be removed (here and in `test_join2()` and `test_join3()`).

Reply:
Updated.